samples/apps/bookinfo/mixer-rule-additional-telemetry.yaml

```diff
@@ -11,7 +11,7 @@ rules:
           source: source.labels["app"] | "unknown"
           target: target.service | "unknown"
           service: target.labels["app"] | "unknown"
-          method: request.path | "unknown"
+          method: request.method | "unknown"
```

Comment:
this was intentional -- method is just a bit overloaded. (it was meant to get at the API method of the service `/ratings` v. the http method `GET`). But I have no strong feelings on the need for it either way.